packages/manager/modules/manager-components/src/region/service.js

```diff
@@ -8,29 +8,32 @@ export default class ovhManagerRegionService {
   }
 
   static getMacroRegion(region) {
-    const localZonePattern = /^lz/i;
-    let macro;
-    if (
-      localZonePattern.test(
-        region
-          .split('-')
-          ?.slice(2)
-          ?.join('-'),
-      )
-    ) {
-      // The pattern for local zone is <geo_location>-LZ-<datacenter>-<letter>
-      // geo_location is EU-WEST, EU-SOUTH, maybe ASIA-WEST in the future
-      // datacenter: MAD, BRU
-      macro = /[\D]{2,3}/.exec(
-        region
-          .split('-')
-          ?.slice(3)
-          ?.join('-'),
-      );
-    } else {
-      macro = /[\D]{2,3}/.exec(region);
-    }
-    return macro ? macro[0].replace('-', '').toUpperCase() : '';
+    /*
+     * Examples for each possible case:
+     *
+     * region: GRA --> macroRegion: GRA
+     * region: GRA5 --> macroRegion: GRA
+     * region: GRA11 --> macroRegion: GRA
+     * region: RBX-ARCHIVE --> RBX
+     * region: CA-EAST-TOR --> macroRegion: TOR
+     * region: CA-EAST-TOR-A --> macroRegion: TOR
+     * region: EU-SOUTH-LZ-MAD --> macroRegion: MAD
+     * region: EU-SOUTH-LZ-MAD-A --> macroRegion: MAD
```

Comment:
There is also

* region: US-EAST-VA-1 --> macroRegion: VA
* region: US-WEST-OR-1 --> macroRegion: OR

It will work because it is the same pattern as `CA-EAST-TOR-A` and will go intro option length `4`.

Reply:
Yes, the two patterns are checked in the same way. No need, for now, to distinguish between a region ending with a letter or a number to extract the macro region.